src/languages/purebasic.js

```diff
@@ -0,0 +1,64 @@
+/*
+Language: PureBASIC
+Author: Tristano Ajmone <tajmone@gmail.com>
+Description: Syntax highlighting for PureBASIC (v.5). No inline ASM highlighting. First release (v.1.0), April 2016.
+Credits: I've taken inspiration from the PureBasic language file for GeSHi, created by Gustavo Julio Fiorenza (GuShH).
+Category: misc
+*/
+
+// Base deafult colors in PB IDE: background: #FFFFDF; foreground: #000000;
+
+function(hljs) {
+	var STRINGS = { // PB IDE color: #0080FF (Azure Radiance)
+		className: 'string',
+		begin: '(~)?"', end: '"',
+		illegal: '\\n',
+		relevance: 0
```

Comment:
No need to zero relevance of strings, it will artificially de-value this language relevance compared to the rest. Just drop the attribute so it gets a default value.